keel-core/src/main/kotlin/com/netflix/spinnaker/keel/lifecycle/MonitoredTask.kt

```diff
@@ -15,6 +15,7 @@ package com.netflix.spinnaker.keel.lifecycle
 data class MonitoredTask(
   val triggeringEvent: LifecycleEvent,
   val link: String,
+  val triggeringEventUid: String,
   val type: LifecycleEventType = triggeringEvent.type,
   val numFailures: Int = 0
```

Comment:
We already store the triggering event uid, let's just return it here so we don't have to look it up again later!